### PR TITLE
chore(IT Wallet): [SIW-1398] Remove local feature flag

### PR DESCRIPTION
**Pull request by hevelius**
## Short description
This PR remove local feature flag. A remote feature flag is used.

## List of changes proposed in this pull request
- Update env local and prod
- Update config and backend

## How to test
Start app also in local dev and check if only remote feature flag is used

### CODE REVIEW

File: ts/boot/configureStoreAndPersistor.ts [hunk 1]
```diff
@@ -422,7 +422,11 @@ const migrations: MigrationManifest = {
   // Version 32
   // Removes new Messages Home FF
   "32": (state: PersistedState) =>
-    omit(state, "persistedPreferences.isNewHomeSectionEnabled")
+    omit(state, "persistedPreferences.isNewHomeSectionEnabled"),
+  // Version 33
+  // Removes it wallet section FF
+  "33": (state: PersistedState) =>
+    omit(state, "persistedPreferences.isItWalletTestEnabled")
 };
```
Comment (LazyAfternoons) [inline ts/boot/configureStoreAndPersistor.ts:430]: For this to work we have to raise the `CURRENT_REDUX_STORE_VERSION` constant.

**Review by LazyAfternoons — CHANGES_REQUESTED** [review]

File: ts/boot/configureStoreAndPersistor.ts (hunk 1 above)
Comment (hevelius) [reply ts/boot/configureStoreAndPersistor.ts:430]: Good catch! Thank you!

**Review by mastro993 — CHANGES_REQUESTED** [review]
Everything works great! I tried to change the remote FF value to false and everything related to IT Wallet is not diasplayed.
However, we have a little issue in the wallet screen: we still display the "Other" section title, which should not displayed if IT Wallet is not enabled.

I think we should add a check in `OtherCardsContainer` component and display the title only if IT Wallet is disabled.

**Review by mastro993 — APPROVED** [review]
LGTM! Previous issue was resolved. Everything is displayed correctly now

**Review by LazyAfternoons — APPROVED** [review]
LGTM, migration happens. The banner disappears when disabling the remote flag on the dev server and it appears when enabling it.

**Pull request merged**